internal/provider/gateway_fqdn.go

```diff
@@ -62,78 +69,81 @@ func NewGatewayFQDNDeployer(ctx context.Context, d *schema.ResourceData, apiClie
 			FQDN:           d.Get("fqdn").(string),
 			TLSPassthrough: d.Get("tls_passthrough").(bool),
 		},
-		ID:               d.Id(),
-		Description:      d.Get("description").(string),
-		Node:             uint32(d.Get("node").(int)),
-		NodeDeploymentID: nodeDeploymentID,
-		APIClient:        apiClient,
-		ncPool:           ncPool,
-		deployer:         deployer.NewDeployer(apiClient.identity, apiClient.twin_id, apiClient.grid_client, ncPool, true, nil, string(deploymentDataStr)),
+		ID:                    d.Id(),
+		Description:           d.Get("description").(string),
+		NodeID:                nodeID,
+		CapacityDeploymentMap: capacityDeploymentMap,
+		CapacityID:            capacityID,
+		APIClient:             apiClient,
+		ncPool:                ncPool,
+		deployer:              deployer.NewDeployer(apiClient.identity, apiClient.twin_id, apiClient.grid_client, ncPool, true, nil, string(deploymentDataStr)),
```

Comment:
why converting deploymentDataStr to str, while it has `Str` in its name?

Reply:
as it's array of bytes